docs/cspell.json

```diff
@@ -0,0 +1,699 @@
+// cSpell Settings
+{
+    // Version of the setting file.  Always 0.2
+    "version": "0.2",
+    // language - current active spelling language
+    "language": "en",
+    // words - list of words to be always considered correct
```

Comment:
Does cspell support comments like this? `json` doesn't support comments in the spec...

Reply:
This is the config Steven has been using—it looks like `cspell` ignores the comments. I can still remove them to prevent weirdness in editors.